templates/config/onpremises-kfd-v1alpha2.yaml.tpl

```
@@ -13,19 +13,20 @@
 apiVersion: kfd.sighup.io/v1alpha2
 kind: OnPremises
 metadata:
-  # The name of the cluster, will be also used as a prefix for all the other resources created
+  # The name of the cluster. It will also be used as a prefix for all the other resources created.
   name: {{.Name}}
 spec:
-  # This value defines which KFD version will be installed and in consequence the Kubernetes version to use to create the cluster,
-  # it supports git tags and branches
+  # Defines which KFD version will be installed and, in consequence, the Kubernetes version used to create the cluster. It supports git tags and branches. Example: v1.30.1.
```

Just to be coherent with the current KFD version
```suggestion
  # Defines which KFD version will be installed and, in consequence, the Kubernetes version used to create the cluster. It supports git tags and branches. Example: v1.29.4.
```

---

I don't think we will change this description on each release, that's why I put random version